tests/simulate/test_simulate.py

```diff
@@ -0,0 +1,120 @@
+"""Unit tests for simulate module."""
+import unittest
+
+import numpy as np
+
+import relentless
+
+class test_Dilute(unittest.TestCase):
+    """Unit tests for relentless.Dilute"""
+
+    #mock functions for use as operations
+    def update_ens(self, ensemble, potentials, options):
+        for o in options:
+            if o=='constant_ens' and options[o]:
+                return
+        ensemble.T += 2.0
+        ensemble.V = relentless.Cube(L=4.0)
+        for t in ensemble.types:
+            ensemble.N[t] += 1.0
+
+    def update_pot(self, ensemble, potentials, options):
+        for o in options:
+            if o=='constant_pot' and options[o]:
+                return
+        for pair in potentials:
+            potentials[pair]['r'] *= 2.0
+            potentials[pair]['u'] *= 3.0
+
+    def test_init(self):
+        """Test creation from data."""
+        operations = [self.update_ens, self.update_pot]
+        options = {'const_ens':True, 'const_pot':True}
```

Comment:
For consistency with above, these should probably be `constant_ens` and `constant_pot`.